ibm/resource_ibm_storage_file.go

```diff
@@ -1230,7 +1230,7 @@ func getSaaSPerformSpacePrice(productItems []datatypes.Product_Item, size int) (
 		}
 
 		keyname := fmt.Sprintf("%d_%d_GBS", capacityMinimum, capacityMaximum)
-		if *item.KeyName != keyname {
+		if !strings.Contains(*item.KeyName, keyname) {
```

Comment:
please add the condition to check item.keyname first not nil and then other condition

if item.KeyName == nil || !strings.Contains(*item.KeyName, keyName) {